its/src/test/java/com/sonar/it/csharp/RazorClassLibProjectTest.java

```diff
@@ -63,4 +63,14 @@ void projectIsAnalyzed() {
 
     assertThat(getComponent(RAZOR_COMPONENT_CLASS_FILE).getName()).isEqualTo("Component.razor");
   }
+
+  @Test
+  void issuesOfS6798AreRaised() {
```

Comment:
At first I followed the same name convention as for the test method added for `S6800`: `issueS6800IsRaised`.

But then I changed to `issuesOfS6798AreRaised`, that seems a better name to me.

If you agree, I would change it for 6800 too, to keep consistency in naming.

Reply:
Yep I think `issuesOfS6798AreRaised` is a bit more accurate.
What do you think @sebastien-marichal ?

Reply:
Good for me!

Reply:
I made the change for S6800 too. This PR is based on the other PR, and it's already rebased on top of its latest `HEAD`, so it should be fine.